debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TestHelper.java

```diff
@@ -217,7 +217,7 @@ public static void dropAllSchemas() throws SQLException {
         TestHelper.execute(dropStmts);
 
         try {
-            TestHelper.executeDDL("init_database.ddl");
+//            TestHelper.executeDDL("init_database.ddl");
```

Comment:
Why is this commented out ?

Reply:
This is just for the test to not go into this step as this takes too much time and not required while test development, will uncomment this before merging.